packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -587,6 +589,17 @@ async function setupOrgBrowser(
 }
 
 export async function activate(extensionContext: vscode.ExtensionContext) {
+  // Set the value of ext.supportedFolders by reading the packages that are
+  // in the sfdx-project.json file.  This property is used by the
+  // Generate Manifest File command, which should be available under all dirs
+  // listed as packages in the project config file
+  const project = SfProject.getInstance(getRootWorkspacePath());
+  const projectDirectories = project.getUniquePackageNames();
+  vscode.commands.executeCommand(
+    'setContext',
+    'ext.supportedFolders',
+    projectDirectories
+  );
```

Comment:
Section of VS Code docs that helped me figure this out:

https://code.visualstudio.com/api/references/when-clause-contexts#in-and-not-in-conditional-operators